gulp/fabric.js

```diff
@@ -517,7 +520,9 @@ gulp.task('component-samples-template', ['build-component-data', 'component-samp
             .on('error', onGulpError)
         .pipe(data(function () {
             var jstag = '';
-            if(typeof storedFiles[folder].script != "undefined") { jstag = storedFiles[folder].script; }
+            var files =  storedFiles[folder]["files"];
+            for(var o=0; o < files.length; o++) {jstag += files[o];}
+            if(typeof storedFiles[folder][folder] != "undefined") { jstag += storedFiles[folder][folder]; }
```

Comment:
Would you mind adding some spaces to the for, if, and brackets?  Also, new lines for the jstag lines, please.  :)